lockfile.json

```
@@ -414,7 +414,7 @@
     "revision": "e01767921df18142055d97407595329d7629e643"
   },
   "perl": {
-    "revision": "79e88f64681660f3961939bf764d8f3b4bbb0d27"
+    "revision": "99bbcb0faddfc79db3fd2996cb56689a4685e78c",
```

```suggestion
    "revision": "99bbcb0faddfc79db3fd2996cb56689a4685e78c"
```

This is what's breaking CI.

---

Gotta love JSON.

---

:facepalm: